go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/veraison/ccatoken v1.0.0
 	github.com/veraison/corim v1.0.0
 	github.com/veraison/dice v0.0.1
-	github.com/veraison/ear v1.0.1
+	github.com/veraison/ear v1.0.2-0.20230424145130-0dee678d30d3
```

Comment:
Please create a realease of the `ear` version (bumping the SemVer as appropriate) and use that rather than a particular hash.

Reply:
Done in [e456186](https://github.com/veraison/services/pull/140/commits/e456186ff16c881d428899fed3d9ee573fe03fec)